yapapi/strategy/base.py

```diff
@@ -86,6 +87,23 @@ async def respond_to_provider_offer(
         """Respond with a modified `DemandBuilder` object to an offer coming from a provider."""
         raise NotImplementedError()
 
+    @abc.abstractmethod
+    async def invoice_accepted_amount(self, invoice: rest.payment.Invoice) -> Decimal:
+        """Return the amount we accept to pay for the invoice.
+
+        Current Golem implementation accepts the invoice if returned amount equals `invoice.amount`
```

Comment:
I wouldn't put this comment here since there's _no_ implementation here

Reply:
I'll add "Golem Engine" and change "equals" to "is not lower than" (as per the prev comment)